TinCan/InteractionType.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace TinCan
+{
+    public sealed class InteractionType
+    {
+        private const string choice = "choice";
+        private const string sequencing = "sequencing";
+        private const string likert = "likert";
+        private const string matching = "matching";
+        private const string performance = "performance";
+        private const string truefalse = "true-false";
+        private const string fillin = "fill-in";
+        private const string numeric = "numeric";
+        private const string other = "other";
```

Comment:
For constants would like to use an all uppercase convention.

Reply:
"long-fill-in" is missing. It is required according to [xAPI Data](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Data.md#object)